impacket/dcerpc/v5/dhcpm.py

```diff
@@ -888,7 +897,7 @@ def hDhcpGetOptionValueV5(dce, option_id, flags=DHCP_FLAGS_OPTION_DEFAULT, class
     request['ClassName'] = classname
     request['VendorName'] = vendorname
     request['ScopeInfo']['ScopeType'] = scopetype
-    request['ScopeInfo']['ScopeInfo']['tag'] = scopetype.value
+    request['ScopeInfo']['ScopeInfo']['tag'] = scopetype
```

Comment:
Once you fix the previous issue, I think you will need to get back to the original commit (i.e. specifying `.value`). 

We will need if the test cases pass tho.

Reply:
In order to fix the test case you should pass scope type as follows
dhcpm.DHCP_OPTION_SCOPE_TYPE.DhcpSubnetOptions
instad of 
dhcpm.DHCP_OPTION_SCOPE_TYPE.enumItems.DhcpSubnetOptions
I don't know if it matters, your choice.
Regardless, the original problem was DHCP_ALL_OPTION_VALUES configured twice so this should be changed anyway.